tools/tutorial_videos/tests/test_tts.py

```
@@ -0,0 +1,147 @@
+            engine, voice="N", style="s", text="hello", cache_dir=self.tmp
+        )
+        stamp = path.stat().st_mtime_ns
+        again = synthesize_cached(
+            engine, voice="N", style="s", text="hello", cache_dir=self.tmp
+        )
+        self.assertEqual(path, again)
+        self.assertEqual(path.stat().st_mtime_ns, stamp)
+        self.assertEqual(len(engine.calls), 1)
+
+    def test_voices_yaml_loads_streams(self):
+        engine_name, model, streams = load_voices(
```

Ruff (0.15.21)

[warning] Unpacked variable `model` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)